lib/src/model/movie_result.dart

```diff
@@ -53,18 +56,29 @@ class MovieResult {
       voteCount: parsedJson['vote_count'],
       id: parsedJson['id'],
       video: parsedJson['video'],
-      voteAverage: parsedJson['vote_average'],
-      title: parsedJson['title'],
+      voteAverage: parsedJson['vote_average'] != null
+          ? parsedJson['vote_average']
+          : MovieDimensions.defaultVoteAverage,
+      title: parsedJson['title'] != null
+          ? parsedJson['title']
+          : MovieStrings.defaultTitle,
       popularity: parsedJson['popularity'],
-      posterPath:
-          parsedJson['poster_path'] != null ? parsedJson['poster_path'] : '',
+      posterPath: parsedJson['poster_path'] != null
+          ? parsedJson['poster_path']
+          : MovieStrings.defaultPosterPath,
       originalLanguage: parsedJson['original_language'],
       originalTitle: parsedJson['original_title'],
       genreIds: _getGenresIdList(genreIds),
       backdropPath: parsedJson['backdrop_path'],
-      adult: parsedJson['adult'],
-      overview: parsedJson['overview'],
-      releaseDate: parsedJson['release_date'],
+      adult: parsedJson['adult'] != null
+          ? parsedJson['adult']
+          : MovieDimensions.defaultIsAdult,
```

Comment:
Here only add false instead of MovieDimensions.defaultIdAdult

Reply:
Done